libnl3.yaml

```diff
@@ -61,6 +61,11 @@ subpackages:
     pipeline:
       - uses: split/dev
 
+  - name: "libnl3-static"
+    description: "static library for libnl3"
+    pipeline:
+      - uses: split/static
```

Comment:
this will be an empty package; as static split must be before split dev. I wonder if split/static should fail if there is nothing to split, or like automatilly also check s|-static|-dev destination for static libraries to slurp up. As this is very error prone.

Reply:
let me fix this, great note btw!